frontend/templates/product/sections/ProductOverviewSection/CompatibleDevices.tsx

```diff
@@ -21,14 +21,16 @@ export function CompatibleDevices({ product }: CompatibleDevicesProps) {
 
    return (
       <>
-         {devices.slice(0, 3).map((device, index) => (
-            <CompatibleDevice
-               key={index}
-               device={device}
-               maxModelLines={MAX_MODEL_LINES_PER_DEVICE}
-               role="group"
-               mb="6px"
-            />
+         {devices.slice(0, MAX_VISIBLE_DEVICES).map((device, index) => (
+            <LinkBox key={index}>
+               <LinkOverlay href="#compatibility" />
```

Comment:
Why the LinkOverlay here?

Reply:
Makes the whole CompatibleDevice card a link. Wrapping it in an anchor causes hydration errors, so this is a workaround.

Reply:
I'd like to see what you mean about wrapping the whole thing in an anchor. Is that with a NextLink or also with a normal a tag?

Reply:
I don't think it matters which link component we use. I think react is complaining that we are wrapping anchor tags in other anchor tags (since CompatibleDevice has anchor tags in it).

```ts
{/* causes hydration errors */}
<a>
   <CompatibleDevice />
</a>
```

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element

> [Transparent](https://html.spec.whatwg.org/multipage/dom.html#transparent), but there must be no [interactive content](https://html.spec.whatwg.org/multipage/dom.html#interactive-content-2) descend[a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)nt, a element descendant, or descendant with the [tabindex](https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex) attribute specified.

Reply:
https://chakra-ui.com/docs/components/link-overlay/usage

Chakra's linkbox works around this specification with zindex it seems

> The LinkBox lifts any nested links to the top using z-index to ensure proper keyboard navigation between links.

Reply:
Ah fair point, though it does raise the question of why we're including a link in CompatibleDevice if we're trying to wrap the whole box in a different link.

Reply:
Good point. I chose to conditionally include the links in e6575c214325d052ea834a5e47d069c95afd6fef, which means we no longer need to use the linkbox too.